e2e/cypress/tests/integration/enterprise/guest_accounts/guest_add_spec.ts

```diff
@@ -13,14 +13,17 @@
 /**
  * Note: This test requires Enterprise license to be uploaded
  */
+import {Channel} from '@mattermost/types/lib/channels';
+import {Team} from '@mattermost/types/lib/teams';
+import {UserProfile} from '@mattermost/types/lib/users';
 
 import {createPrivateChannel} from '../elasticsearch_autocomplete/helpers';
 import * as TIMEOUTS from '../../../fixtures/timeouts';
 
 describe('Guest Account - Guest User Experience', () => {
-    let guestUser;
-    let privateChannel;
-    let testTeam;
+    let guestUser: UserProfile;
+    let privateChannel: Channel;
+    let testTeam: Team;
```

Comment:
Please use the types we have at [here](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/tests/support/index.d.ts) so that we maintain imports in one place only.

Please do the same with others.

```javascript
let guestUser: Cypress.UserProfile;
let privateChannel: Cypress.Channel;
let testTeam: Cypress.Team;
```